src/Aspire.Hosting.Azure.Functions/Aspire.Hosting.Azure.Functions.csproj

```diff
@@ -28,6 +28,7 @@
     </Compile>
     <EmbeddedResource Include="..\Aspire.Hosting\Resources\LaunchProfileStrings.resx">
       <Link>Resources\LaunchProfileStrings.resx</Link>
+      <LogicalName>Aspire.Hosting.Resources.LaunchProfileStrings.resources</LogicalName>
```

Comment:
Is this change intentional?

Reply:
Yep, see https://github.com/dotnet/aspire/pull/12658#issuecomment-3493993102.